devices/pcie/gpus.json

```diff
@@ -107,40 +297,85 @@
         "interface": "PCIe",
         "memory_size": "8Gi"
       },
-      "1bb3": {
-        "name": "p4",
+      "2488": {
+        "name": "rtx3070",
         "interface": "PCIe",
         "memory_size": "8Gi"
       },
-      "1b38": {
-        "name": "p40",
+      "2216": {
+        "name": "rtx3080",
+        "interface": "PCIe",
+        "memory_size": "10Gi"
+      },
+      "2206": {
```

Comment:
@88plug  The last one:

`10de:2206` is `10Gi` RTX 3080, not `12Gi`.

Please update it.